tests/data/author/governed_folders/template_folder_with_template_type/architecture.md

```diff
@@ -0,0 +1,13 @@
+---
+authors: tmp
+owner: tmp
+valid:
+  from: null
+  to: null
+x-trestle-template-type: architecture
+---
```

Comment:
x-trestle-template-type has to be removed from the template

Reply:
done